andela-obamidele/headlines
# Feedback Implementation

#### What does this PR do?

- add more detail to documentation especially in the aspect of contribution
- fix variable and file names that do not follow  Andela naming convention
- add a description to all JSDoc
- remove source link when a user is not logged in 
- make sources search bar more intuitive
- format publish date on article component
- change url to reflect that user is logged out
- create separate webpack for production
#### Description of Task to be completed?
Not Applicable
#### How should this be manually tested?
For changes that is visual, visit http://aproko-international.herokuapp.com
or see screenshots below
for non-visual changes check committed changes
#### Any background context you want to provide?
Not Applicable
#### What are the relevant pivotal tracker stories?
Not applicable
#### Screenshots (if appropriate)
** Date formated well on Articles **
<img width="1392" alt="screen shot 2017-07-01 at 7 15 24 pm" src="https://user-images.githubusercontent.com/29209147/27769759-24f356a2-5f29-11e7-92fa-7e9bec8faeed.png">

** The new searchbar **
<img width="1280" alt="screen shot 2017-07-01 at 3 41 16 pm" src="https://user-images.githubusercontent.com/29209147/27769769-41095a80-5f29-11e7-9431-e11123c02a13.png">

** Sources Link removed from navbar when user is not logged in
<img width="1280" alt="screen shot 2017-06-30 at 2 45 37 pm" src="https://user-images.githubusercontent.com/29209147/27769772-4f97bb3c-5f29-11e7-9fe3-ef7896ca9fb2.png">


#### Questions:
